tidy3d/components/device/sim_data.py

```diff
@@ -255,15 +270,37 @@ def plot_field(
         # select the cross section data
         interp_kwarg = {"xyz"[axis]: position}
         # plot the simulation heat conductivity
-        ax = self.simulation.scene.plot_structures_heat_conductivity(
-            cbar=False,
-            alpha=structures_alpha,
-            ax=ax,
-            **interp_kwarg,
-        )
+        if property_to_plot is not None:
```

Comment:
what if it is `None`? shouldn't we just error?

Reply:
If it's None the field (temperature/voltage) will be plotted without the superimposing electric/heat conductivity. This, however, cannot happen since a `DataError` is raised before it gets here.